ngraph/test/visitors/op/atan.cpp

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "unary_ops.hpp"
+
+using Types = ::testing::Types<UnaryOperatorType<ngraph::op::v0::Atan, element::f32>>;
+
+INSTANTIATE_TYPED_TEST_CASE_P(visitor_without_attribute,
```

Comment:
Please rebase your PR to the latest master and use `INSTANTIATE_TYPED_TEST_SUITE_P` macro

Reply:
done rebase, use new macro for the test